updatesnap/SnapVersionModule/snap_version_module.py

```diff
@@ -62,6 +62,32 @@ def process_snap_version_data(upstreamversion, snap_name, version_schema, has_up
     return f"{upstreamversion}-{packagerelease}"
 
 
+def process_rock_version_data(upstreamversion, prevversion, version_schema, has_update):
+    """ Returns processed rock version"""
+
+    match = re.match(version_schema, upstreamversion)
+    if not match:
+        logging.warning("Version schema does not match with rock repository version")
+        return None
+    upstreamversion = match.group(1).replace('_', '.')
+
+    def version_tuple(v):
```

Comment:
Do we really need to create a function just for this?

Reply:
Removed function and now using inline tuple to resolve version